app/views/customers/_product_results.html.erb

```diff
@@ -0,0 +1,12 @@
+<div>
+  <section class="min-h-full bg-blue-100 lg:p-32 p-14 text-right" style="background-image: url('https://images.unsplash.com/photo-1606836483111-e08386681a8b?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80'); background-size: cover">
+    <h1 class="lg:text-5xl text-xl font-bold text-white mb-14 px-2"><%= params[:name] %></h1>
```

Comment:
It's better to create a variable, in customers controller you can create a variable called *product*, pass it to ProductsQuery class and use it here